src/layers/VolumeLayer/VolumeLayer.js

```diff
@@ -27,9 +28,9 @@ const defaultProps = {
     ],
     compare: true
   },
-  xSlice: { type: 'array', value: [0, 1], compare: true },
-  ySlice: { type: 'array', value: [0, 1], compare: true },
-  zSlice: { type: 'array', value: [0, 1], compare: true },
+  xSlice: { type: 'array', value: null, compare: true },
+  ySlice: { type: 'array', value: null, compare: true },
+  zSlice: { type: 'array', value: null, compare: true },
```

Comment:
Why this change to `null`? The `type` is still "array".

Reply:
Perhaps you have a better way of handling this but because we have to [scale the bounds of these down to 0-1 before passing to the shader](https://github.com/hms-dbmi/viv/pull/422/files#diff-ff964aec2fa91871b0d82b74dacca26597f5a85300adc55ac0600986924f2167R281-R295) providing defaults is difficult.  You would need to know the size of the volume ahead of time, I think, and could not just check whether or not it was `[0,1]` (because you might want the `[0,1]` non-normalized coordinates which then need be normalized).